apps/help-with-fees/help-with-fees-staffapp/demo.yaml

```diff
@@ -7,3 +7,4 @@ spec:
     base:
       environment:
         Trig_1: Demo1
+      image: hmctspublic.azurecr.io/help-with-fees/staffapp:feature/UCD-work-mvp #{"$imagepolicy": "flux-system:demo-help-with-fees-staffapp"}
```

Comment:
this won't work the image will be a pr-ddd image:
https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment-v2.md#deploy-non-prod-image-to-an-environment